checks/http/sap.rb

```diff
@@ -36,10 +36,10 @@ def generate_checks(url)
               ],
               version: nil,
               match_type: :content_headers,
-              match_content: %r{server: SAP NetWeaver ApplicationServer [\d.\s]*/ ICM [\d.]+}i,
+              match_content: %r{server: SAP NetWeaver Application Server [\d.\s]*/ ICM [\d.]+}i,
```

Comment:
same here, fingerprint broken?

Reply:
This was broken in a previous commit I did, so I adjusted them back.